src/Compilers/CSharp/Portable/Emitter/Model/PEModuleBuilder.cs

```diff
@@ -1215,6 +1215,10 @@ internal sealed override Cci.IMethodReference Translate(MethodSymbol symbol, Dia
 
                     return methodRef;
                 }
+                else if (methodSymbol is NativeIntegerMethodSymbol { UnderlyingMethod: MethodSymbol underlyingMethod })
```

Comment:
>else if (methodSymbol is NativeIntegerMethodSymbol { UnderlyingMethod: MethodSymbol underlyingMethod }) [](start = 16, length = 103)

Please make sure we have a test for an expression lambda using the method, including the case when the method is generic.

Reply:
`NativeIntegerTests.ReadOnlyField_VirtualMethods`

---
In reply to: [418299612](https://github.com/dotnet/roslyn/pull/43766#discussion_r418299612) [](ancestors = 418299612)